app-editors/featherpad/featherpad-0.9.1.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="A lightweight Qt5 plain-text editor for Linux"
+DESCRIPTION="FeatherPad (by Pedram Pourang, a.k.a. Tsu Jan \
+<tsujan2000@gmail.com>) is a lightweight Qt5 plain-text editor for Linux \
+It is independent of any desktop environment and has: \
+* Drag-and-drop support, including tab detachment and attachment;
+* X11 virtual desktop awareness (using tabs on current desktop but opening
```

Comment:
Thanks @khallebal for your PR. I did not test the recipe but it looks OK, with just a tiny change: adding a ` \` here.
```diff
- * X11 virtual desktop awareness (using tabs on current desktop but opening
+ * X11 virtual desktop awareness (using tabs on current desktop but opening \
  a new window on another);
```